products/cloudtasks/api.yaml

```diff
@@ -170,6 +170,18 @@ objects:
             name: 'purgeTime'
             output: true
             description: The last time this queue was purged.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'stackdriverLoggingConfig'
+        description: |
+          Configuration options for writing logs to Stackdriver Logging.
+        properties:
+          - !ruby/object:Api::Type::Double
+            name: 'samplingRatio'
```

Comment:
We should make this subfield required, to avoid people specifying `stackdriver_logging_config {}`

We generally try to avoid having blocks with no required fields (atleastoneof or exactlyoneof are fine as well)

Reply:
done